tests/tests.py

```diff
@@ -230,6 +229,7 @@ def test_single_form_csv_with_rep_fields_start_with_repeat(self):
         for line1, line2 in izip(open(get_filename('models.py'),'r'),open(get_filename(cmp_fileName),'r')):
             self.assertEqual(line1,line2)
     def test_single_form_csv_with_rep_fields_many_nested(self):
```

Comment:
It is hard to say what else is missing from the test file, and what the header of the file should be. A new test should be made or the file fixed, but for now the test is ignored in order to see if the rest of the issue-19 branch is functional.